[EDU-474] Data Studios doc audit (#407)
seqeralabs/docs: platform_versioned_docs/version-24.3/studios/index.mdx

## Container image templates

> There are four container image templates provided: JupyterLab, RStudio Server, Visual Studio Code, and Xpra. The image templates install a very limited number of packages when the Studio session container is built. You can install additional packages as needed during a Studio session.

Reviewer: How about:

```
You can install additional packages as needed during a studio session.
```

- `nb_black=1.0.7`
- `qgrid=1.3.1`

To install additional Python packages during a running Studio session, execute `!pip install <packagename>` commands in your notebook environment. Additional system-level packages can be installed in a terminal window using `apt install <packagename>`.

To see the list of all JupyterLab image templates available, see [public.cr.seqera.io/repo/platform/data-studio-jupyter][ds-jupyter].

**RStudio Server 4.4.1**

The default user is the `root` account. To install R packages during a running Studio session, execute `install.packages("<packagename>")` commands in your notebook environment. Additional system-level packages can be installed in a terminal window using `apt install <packagename>`.

To see the list of all RStudio Server image templates available, see [public.cr.seqera.io/repo/platform/data-studio-rstudio][ds-rstudio].

**Visual Studio Code 1.93.1**

> [Visual Studio Code][def-vsc] is an integrated development environment (IDE) that supports many programming languages. The default user is the `root` account. The container template image ships with the latest stable version of [Nextflow] and the [VSCode extension for Nextflow][nf-lang-server] to make troubleshooting Nextflow workflows easier. To install additional extensions during a running Studio session, select **Extensions**. Additional system-level packages can be installed in a terminal window using `apt install <packagename>`.

Reviewer 2: ```suggestion
[Visual Studio Code][def-vsc] is an integrated development environment (IDE) that supports many programming languages. The default user is the `root` account. The container template image ships with the latest stable version of [Nextflow] and the [VS Code extension for Nextflow][nf-lang-server] to make troubleshooting Nextflow workflows easier. To install additional extensions during a running Studio session, select **Extensions**. Additional system-level packages can be installed in a terminal window using `apt install <packagename>`.
```

To see the list of all Visual Studio Code image templates available, see [public.cr.seqera.io/platform/data-studio-vscode][ds-vscode].

**Xpra 6.2.0**

[Xpra][def-xpra], known as _screen for X_, allows you to run X11 programs by giving you remote access to individual graphical applications. The container template image also installs NVIDIA Linux x64 (AMD64/EM64T) drivers for Ubuntu 22.04 for running GPU-enabled applications. To use these GPU drivers, your compute environment must specify GPU instance families.

The default user is the `root` account. The image is based on `ubuntu:jammy`. Additional system-level packages can be installed during a running Studio session in a terminal window using `apt install <package_name>`.

To see the list of all Xpra image templates available, see [public.cr.seqera.io/repo/platform/data-studio-xpra][ds-xpra].

> ## Studio session statuses

Author: Note: This section should be reviewed for language and cadence.

Studios have the following possible statuses:

- **building**: When a custom environment is building the template image for a new Studio session. The [Wave] service performs the build action. For more information on this status, see [Inspect custom container template build status][build-status].
- **build-failed**:  When a custom environment build has failed. This is a non-recoverable error. Logs are provided to assist with troubleshooting. For more information on this status, see [Inspect custom container template build status][build-status].
- **starting**: The Studio is initializing.
- **running**: When a Studio session is **running**, you can connect to it, copy the URL, or stop it. In addition, the session can continue to process requests/run computations in the absence of an ongoing connection.
- **stopping**: The recently-running session is in the process of being stopped.

> - **stopped**: When a session is stopped, the associated compute resources are deallocated. You can start or delete the Studio when it's in this state.

Author: ```suggestion
- **stopped**: When a Studio session is stopped, the associated compute resources are deallocated. You can start or delete the session when it's in this state.
```

> - **errored**: This state most often indicates that there has been an error starting the Studio session but it is in a **stopped** state. There might be errors reported by the session itself but these will be overwritten with a **running** status if the session is still running.

Author: @robnewman, this doesn't seem correct. 

If there's an error that results in a stopped state, what relevance does the second sentence have? Should it be a separate point?

Reviewer: > @robnewman, this doesn't seem correct.
> 
> If there's an error that results in a stopped state, what relevance does the second sentence have? Should it be a separate point?

Yes - perhaps move to an admonishment.

  :::tip
  If you encounter an error with the public preview release of Studios, [contact Seqera support][contact].
  :::

## Studio session data links

> You can configure a Studio to mount one or more data links, where cloud buckets that you have configured in your compute environment are read-only, or read-write available to the session.

Author: ```suggestion
You can configure a Studio session to mount one or more data links, where cloud buckets that you have configured in your compute environment are read-only, or read-write available to the session.
```

Reviewer: @justinegeffen I thought we agreed it was either "Studios" (proper noun) or "studio". In this case, shouldn't it be `studio session` and not `Studio session`? I guess it could be `Studios session`, but that doesn't feel right.

To see the list of all Xpra image templates available, see [public.cr.seqera.io/repo/platform/data-studio-xpra][ds-xpra].

## Studio session statuses

> Studios have the following possible statuses:

Reviewer: ```suggestion
Sessions have the following possible statuses:
```

If the maximum allocation size is reached, it is possible to reclaim storage space using a snapshot. 

> Stop the active session to trigger a snapshot from the active volume. Studios uploads the snapshot to cloud storage with Fusion. When you start from the newly saved snapshot, all previous data is loaded and the newly-started session will have 2 GB of available space.

Author: ```suggestion
Stop the active Studio session to trigger a snapshot from the active volume. The snapshot is uploaded to cloud storage with Fusion. When you start from the newly saved snapshot, all previous data is loaded, and the newly started session will have 2 GB of available space.
```
